lib/tasks/instrumentation_generator/templates/Envfile.tt

```diff
@@ -0,0 +1,5 @@
+instrumentation_methods :chain, :prepend
```

Comment:
needs the frozen string literal magic comment

Reply:
I took a crack at getting the templates linted automatically with:

```yaml
AllCops:
  Include:
    - 'lib/tasks/**/templates/*.tt'
```

only to be reminded that RuboCop still can't handle stumbling upon an ERB tag inside of a file. 😞

I suppose for this PR's scope (the templates are in Ruby and render even more Ruby!), that inner layer of Ruby wouldn't be getting linted anyhow.

Reply:
Literal comment added in: 7b628d969328a75e5059dd59529eb763958bd8ff